mechanics/physicell/tests/test_mechanical_agent_container.cpp

```diff
@@ -0,0 +1,110 @@
+#include <gtest/gtest.h>
+
+#include "physicell/mechanical_agent_container.h"
+
+namespace physicore::mechanics::physicell::tests {
+
+using namespace physicore;
+using namespace physicore::mechanics::physicell;
+
+namespace {
+mechanical_agent_container make_mechanical_agent_container()
```

Comment:
all the tests here add zero coverage - everything that you have tested here has already been tested in common/tests.
Rewrite this whole file as is in `biofvm/tests/test_agent_container.cpp`. the most important coverage  we want to get from this file is in RemoveAgentsAndCheckProperties. You (1) create multiple agents, (2) set all the properties different values, (3) then you remove an agent and test that all the remaining agents have their properties unchanged.